src/llfs/packed_log_control_block2.hpp

```diff
@@ -0,0 +1,81 @@
+//#=##=##=#==#=#==#===#+==#+==========+==+=+=+=+=+=++=+++=+++++=-++++=-+++++++++++
+//
+// Part of the LLFS Project, under Apache License v2.0.
+// See https://www.apache.org/licenses/LICENSE-2.0 for license information.
+// SPDX short identifier: Apache-2.0
+//
+//+++++++++++-+-+--+----- --- -- -  -  -   -
+
+#pragma once
+#ifndef LLFS_PACKED_LOG_CONTROL_BLOCK2_HPP
+#define LLFS_PACKED_LOG_CONTROL_BLOCK2_HPP
+
+#include <llfs/config.hpp>
+//
+#include <llfs/int_types.hpp>
+#include <llfs/packed_slot_offset.hpp>
+
+#include <array>
+
+#include <batteries/static_assert.hpp>
+
+namespace llfs {
+
+struct PackedLogControlBlock2 {
+  static constexpr u64 kMagic = 0x128095f84cfba8b0ull;
```

Comment:
Magic numbers are sanity checks -- they are randomly selected byte patterns that occur at the start of certain packed structures.  They can be used to validate that the expected sort of data was indeed likely stored at this location, and can also be useful during data recovery when it's not clear where things are, because some config information is missing or corrupted.

Reply:
That makes a lot of sense! Nifty.